src/binding.cc

```diff
@@ -673,6 +673,11 @@ const v8::Boolean* v8__Value__ToBoolean(const v8::Value& self,
   return local_to_ptr(self.ToBoolean(isolate));
 }
 
+bool v8__Value__InstanceOf(const v8::Value& self, const v8::Context& context,
```

Comment:
```suggestion
bool v8__Value__InstanceOf(v8::Value& self, const v8::Context& context,
```
And `*mut Value` in rust land.